mdpow/workflows/dihedrals.py

```diff
@@ -459,25 +528,24 @@ def dihedral_violins(df, width=0.9, solvents=SOLVENTS_DEFAULT):
                             "interaction",
                             "lambda"]).reset_index(drop=True)
 
-    # number of Coul windows + 1 / number of VDW windows
-    # (+1 for additional space with axes)
     width_ratios = [len(df[df['interaction'] == "Coulomb"]["lambda"].unique()) + 1,
-                    len(df[df['interaction'] == "VDW"]["lambda"].unique())]
+                    len(df[df['interaction'] == "VDW"]["lambda"].unique()),
+                    len(df[df['interaction'] == "Coulomb"]["lambda"].unique()) - 1]
 
-    solvs = np.asarray(solvents)
-    solv2 = 'octanol'
-    if solvs.size > 1:
-        solv2 = solvs[1]
+    solvs = list(solvents)
```

Comment:
I think we don't need to convert to a list if we use in catplot
```python
hue_order=solvents[:2], ...
```
If catplot _really_ needs a list, then we can use
```python
hue_order=list(solvents[:2]), ...
```

See below at catplot.